db/migrate/20210405143721_add_reading_list_document_to_articles.rb

```diff
@@ -0,0 +1,6 @@
+class AddReadingListDocumentToArticles < ActiveRecord::Migration[6.0]
+  def change
+    add_column :articles, :reading_list_document, :tsvector
```

Comment:
> Huh, I was under the impression that the pg gem didn't know how to deserialize tsvectors off the wire at all and warned that they were just being deserialized as strings.

I vaguely remember that. It might be that Rails added support for `tsvector` or `pg` did. 

> With that in mind, would it be worth adding it to ignored_columns so we don't pull it into the app?

Don't know, I'm conflicted, if we ignore, it'll mean we can't test the trigger's behavior. Well, we can still fetch it via PG and manual SQL queries. What do you think?

Reply:
Ah, that's a good point. Might be worth leaving it in for now and revisit later.

It's not a big deal either way, tbh. I'm mainly thinking about how we have no use for the attribute in Ruby code, and since `articles` is a pretty wide table (`article.attributes.count` shows 74 columns), there is CPU time spent in deserializing all these columns, and it's one of our god objects so we're probably deserializing a lot of data we don't need _all the time_, we might be able to make things a bit faster if we leave out all the columns we're only using inside queries.

I don't have any numbers on any of this, though. It's just some speculation that started when I learned about `ignore_columns` last week. :joy: As much as I'd like to get benchmarks on it I really don't think it's a blocker for this PR. I'll likely experiment with the idea at some point — not just this column or even this table, but the `users` table is pretty wide, too.